### ISSUE TITLE
<details> and <summary> not supported by Azure DevOps

### ISSUE THREAD

**Issue by MaxMommersteeg** [NuKeeperDotNet/NuKeeper#709]
Currently used tags in pull request descriptions like `<details></details>` and `<summary></summary>` are rendered incorrectly by Azure DevOps since they are unsupported and platform specific.

How it displays in Azure DevOps:
```
2 packages were updated in 1 project:
Microsoft.ApplicationInsights.AspNetCore, Microsoft.ServiceFabric.AspNetCore.HttpSys
<details>
<summary>Details of updated packages</summary>

NuKeeper has generated a minor update of Microsoft.ApplicationInsights.AspNetCore to 2.6.1 from 2.5.1
Microsoft.ApplicationInsights.AspNetCore 2.6.1 was published at 2019-02-08T00:39:23Z, 2 months ago

1 project update:
Updated src\Services.ApiGateway\Services.ApiGateway.csproj to Microsoft.ApplicationInsights.AspNetCore 2.6.1 from 2.5.1

Microsoft.ApplicationInsights.AspNetCore 2.6.1 on NuGet.org

NuKeeper has generated a minor update of Microsoft.ServiceFabric.AspNetCore.HttpSys to 3.3.644 from 3.2.162
Microsoft.ServiceFabric.AspNetCore.HttpSys 3.3.644 was published at 2019-03-12T19:13:06Z, 19 days ago

1 project update:
Updated src\Services.ApiGateway\Services.ApiGateway.csproj to Microsoft.ServiceFabric.AspNetCore.HttpSys 3.3.644 from 3.2.162

Microsoft.ServiceFabric.AspNetCore.HttpSys 3.3.644 on NuGet.org

</details>

This is an automated update. Merge only if it passes tests
NuKeeper: https://github.com/NuKeeperDotNet/NuKeeper
```

This issue can probably be resolved with pull request #665 or a pull request that builds upon that code. Added this issue for convenience and traceability.

**Comment by AnthonySteele** [comment]
Because of issues like this, the commit wording will have to be platform-specific, Maybe with mostly common parts.

**Comment by MaxMommersteeg** [comment]
Solved with pull request #737.
